manifests/init.pp

```diff
@@ -54,6 +55,7 @@
   Optional[Hash] $permissive      = undef,
   Optional[Hash] $port            = undef,
   Optional[Hash] $exec_restorecon = undef,
+  Optional[Hash] $login           = undef,
```

Comment:
please switch this to:
```suggestion
  Hash[String[1],Hash[String[1],String[1]]] $login = {},
```

This should allow an empty hash or an hash with a string as key and another hash as value